libraries/apollo-api/src/commonMain/kotlin/com/apollographql/apollo3/api/http/DefaultHttpRequestComposer.kt

```diff
@@ -36,11 +36,16 @@ class DefaultHttpRequestComposer(
     val operation = apolloRequest.operation
     val customScalarAdapters = apolloRequest.executionContext[CustomScalarAdapters] ?: CustomScalarAdapters.Empty
 
-    val requestHeaders = listOf(
-        HttpHeader(HEADER_APOLLO_OPERATION_ID, operation.id()),
-        HttpHeader(HEADER_APOLLO_OPERATION_NAME, operation.name()),
-        HttpHeader(HEADER_ACCEPT_NAME, HEADER_ACCEPT_VALUE),
-    ) + (apolloRequest.httpHeaders ?: emptyList())
+    val requestHeaders = mutableListOf<HttpHeader>().apply {
```

Comment:
You probably know it already but just in case - there's `buildList {}` :)

Reply:
Nope, we're compatible with very old kotlin-stdlib. We can't have nice things!

Reply:
😞  Oh well 😅